/physics_sim:
  ros__parameters:
    update_rate: 0.02

/force_pub_sub:
  ros__parameters:
    update_rate: 0.02
  
/mpc:
  ros__parameters:
    update_rate: 0.02